src/api/handlers/PetInsertHandler.ts

```diff
@@ -10,13 +9,13 @@ class PetInsertHandler {
     }
 
     public execute(event: any) {
-        const pets = this.service.insert(event.body);
+        const pets = this.service.insert(JSON.parse(event.body));
```

Comment:
@bigLucas 
I had a problem with `pet.id` because the content got by `event.body` returns a _string_ .
To solve this, I converted using `JSON.parse` . Is it correct? 
If so, can I keep the conversion as a parameter or do I need to convert it out first?

Reply:
Yes, it's correct.